table.cc

```diff
@@ -1220,14 +1222,6 @@ table::on_compaction_completion(const std::vector<sstables::shared_sstable>& new
             });
             _sstables_compacted_but_not_deleted.erase(e, _sstables_compacted_but_not_deleted.end());
             rebuild_statistics();
```

Comment:
Does _sstables_compacted_but_not_deleted make any sense after this patch?

Reply:
@raphaelsc ^^^

Reply:
@avikivity
1. I think we still want operations like `sstable_info` to remain conservative and account for these sstables until they are deleted.
2. `get_fully_expired_sstables` algorithm takes `compacted_undeleted_sstables` into account so removing it entirely would be quite intrusive.